### PR TITLE
Add Open mHealth identifier

### PR DESCRIPTION
**Pull request by emersonf**
A new `openmhealth` identifier to give [Open mHealth](https://openmhealth.org/) schemas a permanent home. The details are in the README.

### CODE REVIEW

**Comment by emersonf** [comment]
I kept this as a draft PR until @scarini can review.

File: omh/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Open mHealth
+THe `omh` [W3ID](https://w3id.org) provides permanent URIs for [Open mHealth](https://www.openmhealth.org), a non-profit making patient-generated data accessible through an open data standard and community.
+
+## Schemas 
+Permanent [base URIs](https://json-schema.org/understanding-json-schema/structuring.html#base-uri) are provided for Open mHealth's [data schemas](https://www.openmhealth.org/documentation/#/schema-docs/overview). A base URI takes the form:
+```text
+https://w3id.org/omh/schemas/<schema_namespace>/<schema_name>-<schema_version>.json
+```
+
+e.g.
```
Comment (TallTed) [inline omh/README.md:10]:
```suggestion
— as in these examples —
```

**Comment by dgarijo** [comment]
@emersonf are you still waiting for review, or shall we merge it?

**Comment by emersonf** [comment]
@dgarijo it's ready to merge, thanks!

**Pull request merged**